src/main/java/io/singularitynet/Client/VereyaModClient.java

```diff
@@ -154,10 +156,11 @@ public InputType getInputType() {
     // Control overriding:
     enum InputType
     {
-        HUMAN, AI
+        HUMAN, AI, HYBRID
     }
 
     protected InputType inputType = InputType.HUMAN;
+    protected InputType inputTypeAbs = InputType.HUMAN;
```

Comment:
what is Abs in inputTypeAbs?

Reply:
I don't know. This naming comes from Oleg. Suggestions?

Reply:
i suggest to remove this variable, current code seems too complicated with three variables input type